BranchMetrics/ios-branch-deep-linking-attribution
# Qentelli test worklow

## Reference

SDK-1874-Run tests from qentelli-saas-sdk-testing-automation repo
https://branch.atlassian.net/browse/SDK-1874

## Summary
Updated file .github/workflows/automation-trigger-test.yml to
* Build and upload static lib which will be used by Qentelli Automation Testbed App (for linking with framework build with the current branch).
* Build Qentelli Automation Testbed App 
* Creat and upload IPA 
* Check out code from qentelli-saas-sdk-testing-automation and copy uploaded IPA to testdata/builds/iOS_TestbedApp_21-Jun-2022-1.ipa
* Run tests and upload report
Note : Last job `buildAndRunAutomationTestSuite:`  is a copy-paste from https://github.com/BranchMetrics/qentelli-saas-sdk-testing-automation/blob/master/.github/workflows/ios-maven.yml. I have used same steps for running tests using maven.

## Motivation
Integrate Qentelli Automation Tests with iOS SDK.

## Type Of Change

## Testing Instructions
Run workflow manually from the current Branch. Please see attached screen shot.
![image](https://user-images.githubusercontent.com/93544270/229188126-5ce1315a-c58b-48b7-b932-b781409cf5c9.png)










cc @BranchMetrics/saas-sdk-devs for visibility.
